resources/assets/lib/beatmap-discussions/review-post.tsx

```diff
@@ -37,16 +44,44 @@ export class ReviewPost extends React.Component<Props> {
             'strong',
             'text',
           ]}
-          plugins={[
-            MarkdownEmbedTokenizer,
-          ]}
-          source={this.props.message}
+          key={osu.uuid()}
+          source={source}
           renderers={{
-            embed: ReviewPostEmbed,
             link: (props) => <a className='beatmap-discussion-review-post__link' rel='nofollow' {...props}/>,
-            paragraph: (props) => <div className='beatmap-discussion-review-post__block' {...props}/>,
+            paragraph: (props) => {
+              return <div className='beatmap-discussion-review-post__block'>
+                <div className='beatmapset-discussion-message' {...props}/>
+              </div>;
+            },
           }}
         />
+    );
+  }
+
+  render() {
+    const docBlocks: JSX.Element[] = [];
+
+    try {
+      const document = JSON.parse(this.props.message);
```

Comment:
looks like it's missing typing. Also not sure if shadowing global `document` is a good idea.